includes/reports/overview/list-table/class-sensei-reports-overview-list-table-courses.php

```diff
@@ -85,10 +86,21 @@ public function get_columns() {
 				esc_html( count( $all_course_ids ) )
 			),
 			'last_activity'      => __( 'Last Activity', 'sensei-lms' ),
+			'enrolled'           => sprintf(
+				// translators: Placeholder value is the total average progress for all courses.
+				__( 'Enrolled (%d)', 'sensei-lms' ),
+				$total_enrolled
+			),
+
 			'completions'        => sprintf(
-			// translators: Placeholder value is the number of completed courses.
-				__( 'Completed (%d)', 'sensei-lms' ),
-				esc_html( $total_completions )
+				// translators: Placeholder value is the % of enrollments with completed course.
+				__( 'Completions (%s)', 'sensei-lms' ),
+				$total_completion
+			),
+			'completion_rate'    => sprintf(
+			// translators: Placeholder value is the % of enrollments with completed course.
```

Comment:
Looks like the same message as 5 lines above :) 
And I see a problem with indentation.

Reply:
Fixed 19ba9b92150eccde57b1fc65ae2575854852764e